core/server/master/src/test/java/alluxio/underfs/UfsStatusCacheTest.java

```diff
@@ -371,18 +372,26 @@ public void testFetchExistingSingleStatus() throws Exception {
   @Test
   public void testFetchSingleStatusNonExistingPath() throws Exception {
     spyUfs();
-    UfsStatus fetched = mCache.fetchStatusIfAbsent(new AlluxioURI("/testFile"), mMountTable);
+    try {
+      UfsStatus fetched = mCache.fetchStatusIfAbsent(new AlluxioURI("/testFile"), mMountTable);
+      fail("Should have thrown FileNotFoundException");
```

Comment:
Can you replace these with`ExpectedException` rule?

Reply:
Using the ExpectedException rule I can't use `Mockito.verify` afterwards